community-containers/dlna/dlna.json

```diff
@@ -0,0 +1,38 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-dlna",
+            "display_name": "DLNA",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/dlna",
+            "image": "thanek/nextcloud-dlna",
+            "image_tag": "latest",
+            "internal_port": "host",
+            "restart": "unless-stopped",
+            "depends_on": [
+                "nextcloud-aio-database",
+                "nextcloud-aio-nextcloud"
```

Comment:
This is not going to work as you might think. It only means that the container will be started after the nextcloud container. However the nextcloud container might still not be ready then since it will first start the installation on the first startup if not already done. So instead you should probably add a logic inside your container that first checks maybe in t he database if nextcloud is actually installed.

Reply:
To be honest, I have no idea how can I check if the nextcloud installation is present or is in the preparation phase, or how the dlna container  should behave when it faces this. So I removed the dependency here and added an entry to the Notes section in the readme of nextcloud-dlna community container.

Reply:
See https://github.com/nextcloud/all-in-one/pull/3614#discussion_r1376419690